lab/lab2/part3-application.html

```diff
@@ -9,11 +9,11 @@
     <!--Left Panel-->
     <div style="position: absolute; left: 0px; width: 400px; top: 0; bottom: 0;">
       <br>
-      Numeric filter: <input id="num1" type="number" name="num1" value=""> to <input id="num2" type="number" name="num2" value="">
+      YearBuilt: <input id="num1" type="number" name="num1" value=""> to <input id="num2" type="number" name="num2" value="">
```

Comment:
![image](https://user-images.githubusercontent.com/8103418/53311544-4ddf6980-387f-11e9-86f1-b40c2aaf4794.png)

This is very clear. Nice work.